language/move-lang/README.md

```diff
@@ -0,0 +1,136 @@
+---
+id: move-lang
+title: Move Source Language
```

Comment:
We could try to make these phrases ergonomic as well.  How about renaming as follows:

Move bytecode language -> Move bytecode
Move source language -> Move language

Reply:
I don't use 'Move bytecode language' anywhere, so agreed there. 

I used move 'Move source language' to make it clear this is extra different from any thing else we tend to slap the word Move on (in other words we tend to use "Move language" when we mean "Move language and runtime and environment"). but I'd be interested to hear other people's thoughts